bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
That doesn't explain why you can't import just { Component } as we do everywhere else, without having React in the same scope.
It looks like this will be fixed soon: https://discuss.reactjs.org/t/es6-import-as-react-vs-import-react/360/3